src/components/ReportsPage/HeatmapPage/LevelsInfoCell.tsx

```diff
@@ -48,15 +48,18 @@ export const LevelsInfoCell = ({ identifier, scores }: LevelsInfoCellProps) => {
           </Typography>
         </AccordionSummary>
         <AccordionDetails style={{ display: 'flex', flexDirection: 'column' }}>
-          {scores?.map(score => (
-            <EntityRefLink
-              key={`LevelService-${identifier}-${score.serviceId}`}
-              entityRef={parseEntityRef(
-                score.componentRef,
-                defaultComponentRefContext,
-              )}
-            />
-          ))}
+          <ul style={{ paddingLeft: 16 }}>
```

Comment:
I found 16px by inspecting source code and lining up the padding. Is there a more canonical way to do this?

Reply:
I think this is okay but @nikhilunni pls confirm

Reply:
yeahh that's okay i think -- idk enough about actual FE best practices, but i would've thought the point of a system like bootstrap is to stick to standard spacing instead of hardcoding? like a class like `ml-{1,2,3,etc}`

Reply:
https://getbootstrap.com/docs/4.0/utilities/spacing/ or wtv the actual appropriate class is supposed to be

Reply:
As discussed in Slack, this is the backstage plugin so no Bootstrap here. However, I switched to use the `makeStyles` function and moved the `16px` declaration into there.